api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -1054,6 +1072,306 @@ public void findSurveyVersionByQuestionConceptId() {
+    dob.setTime(tomorrowDob.getDob());
+    assertEquals(-1, (today.get(Calendar.DAY_OF_YEAR) - dob.get(Calendar.DAY_OF_YEAR)));
+    // save DbPerson entities
+    personDao.save(todayDob);
+    personDao.save(yesterdayDob);
+    personDao.save(tomorrowDob);
+
+    List<AgeTypeCount> expected = new ArrayList<>();
+    // preserve order of output - order by age, count
+    expected.add(new AgeTypeCount().ageType("AGE").age(age - 1).count(1l)); // birthday tomorrow
+    expected.add(
+        new AgeTypeCount().ageType("AGE").age(age).count(2l)); // birthday yesterday and today
+    expected.add(new AgeTypeCount().ageType("AGE_AT_CDR").age(20).count(3l));
+    expected.add(new AgeTypeCount().ageType("AGE_AT_CONSENT").age(18).count(3l));
+
+    List<AgeTypeCount> response =
+        controller.findAgeTypeCounts(WORKSPACE_NAMESPACE, WORKSPACE_ID).getBody().getItems();
+    assertIterableEquals(expected, response);
+  }
+
+  @Test
+  public void findDomainCountByStandardSource() {
+    // standardCriteria
+    for (int i = 0; i < 10; i++) {
+      cbCriteriaDao.save(
+          DbCriteria.builder()
+              .addDomainId(Domain.CONDITION.toString())
+              .addType(CriteriaType.SNOMED.toString())
+              .addCount(100L + i)
+              .addHierarchy(true)
+              .addConceptId(String.valueOf(1000 + i))
+              .addStandard(true)
+              .addSelectable(true)
+              .addCode("120")
+              .addName("Diabetes 1")
+              .addFullText("Diabetes 1[CONDITION_rank1]")
+              .build());
+    }
+    // sourceCriteria
+    cbCriteriaDao.save(
+        DbCriteria.builder()
+            .addDomainId(Domain.CONDITION.toString())
+            .addType(CriteriaType.ICD9CM.toString())
+            .addCount(200L)
+            .addHierarchy(false)
+            .addConceptId("13000")
+            .addStandard(false)
+            .addSelectable(true)
+            .addCode("130")
+            .addName("Other liver")
+            .addFullText("Other liver[CONDITION_rank1]")
+            .build());
+
+    DomainCount domainCountStd =
+        controller
+            .findDomainCountByStandardSource(
+                WORKSPACE_NAMESPACE, WORKSPACE_ID, Domain.CONDITION.toString(), Boolean.TRUE, "120")
+            .getBody();
+    assertThat(domainCountStd.getConceptCount()).isEqualTo(10);
+
+    DomainCount domainCountSrc =
+        controller
+            .findDomainCountByStandardSource(
+                WORKSPACE_NAMESPACE,
+                WORKSPACE_ID,
+                Domain.CONDITION.toString(),
+                Boolean.FALSE,
+                "130")
+            .getBody();
+    assertThat(domainCountSrc.getConceptCount()).isEqualTo(1);
+  }
+
+  @Test
+  public void validateDomainCaseInsensitive() {
+    List<String> values =
+        Stream.of(Domain.values()).map(Domain::toString).collect(Collectors.toList());
+    for (String value : values) {
+      assertDoesNotThrow(
```

Comment:
It is probably ok to leave this assertion. I'm not positive but not sure if google Truth supports this.